### PR TITLE
Removing duplicate entries in participant selector

### PR DESCRIPTION
**Pull request by bmuniswamy**
# Critical Changes

# Changes
Filtering out the availableRows array before adding the deselected value to avoid duplicate values
# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
~~- [ ] Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage~~  
~~- [ ] CRUD/FLS is enforced in Apex~~
~~- [ ] Permission sets are updated to account for CRUD|FLS|Tab|Classes~~
~~- [ ] Field sets are updated to account for new fields~~
~~- [ ] UX approval or UX not necessary~~
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [x] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [x] PR contains draft release notes
- [ ] QE story level testing completed

### CODE REVIEW

File: force-app/main/default/lwc/participantSelector/participantSelector.js [hunk 1]
```diff
@@ -371,6 +371,11 @@ export default class ParticipantSelector extends LightningElement {
 
     handleDeselectParticipant(event) {
         if (event) {
+            //filter availableEngagementRows to remove program engagement that we deselected so there are no duplicates
+            this.availableEngagementRows = this.availableEngagementRows.filter(
```
Comment (ptewson-sfdo) [inline force-app/main/default/lwc/participantSelector/participantSelector.js:375]: If I'm reading this right, this is basically just preventing duplicates - here on line 374-378 you are removing any availableEngagementRows that match the deselected participant, and then down in 389-392 you are adding the deselected participant to the availableEngagementRows list from the event.detail.row fired from the deselected participant?

Comment (bmuniswamy) [reply force-app/main/default/lwc/participantSelector/participantSelector.js:375]: Yeah that is correct @ptewson-sfdo

**Review by ptewson-sfdo — APPROVED** [review]
Looks good!

**Comment by bmuniswamy** [comment]
[W-9933291](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000009WjGDYA0/view)

**Pull request merged**